app/views/submissions/steps/trial/_step02.html.haml

```diff
@@ -1,17 +1,17 @@
 .alert.alert-info
   Before uploading your trait score sheet, please choose
-  %b plant traits
+  %b existing trait descriptors
   that were scored in the course of this trial,
   or define
   %b new trait descriptors.
   Further instructions are found below.
-  Examples of plant traits in the database are: 'oleic acid content' or ' seed mature time'
+  Examples of plant traits in the database are: 'oleic acid content' or ' seed mature time'.
 
 .form-group
   = content.label :trait_descriptor_list
-  - options = options_for_submission_select(@content.existing_trait_descriptors, :descriptor_name)
-  - options.concat options_for_submission_select(@content.new_trait_descriptors, :descriptor_name, id: false)
-  = content.select :trait_descriptor_list, options.html_safe, {},
+  -#- options = options_for_submission_select(@content.existing_trait_descriptors, :descriptor_name)
+  -#- options.concat options_for_submission_select(@content.new_trait_descriptors, :descriptor_name, id: false)
```

Comment:
I see, thanks for explaining that to me. The issue might be due to the change in the following line - it does not use the `options` variable anymore. Can you test after reverting the change in the following line?